--- src/app/spartacus/spartacus-configuration.module.ts	2021-07-16 09:54:00.000000000 +0200
+++ src/app/spartacus/spartacus-configuration.module.ts	2021-07-16 14:55:25.000000000 +0200
@@ -2,18 +2,28 @@
 import { translationChunksConfig, translations } from "@spartacus/assets";
 import { FeaturesConfig, I18nConfig, OccConfig, provideConfig, SiteContextConfig } from "@spartacus/core";
 import { defaultCmsContentProviders, layoutConfig, mediaConfig } from "@spartacus/storefront";
+import { environment } from 'src/environments/environment';
+
+const occConfig: OccConfig = { backend: { occ: {} } };
+// only provide the `occ.baseUrl` key if it is explicitly configured, otherwise the value of
+// <meta name="occ-backend-base-url" > is ignored.
+// This in turn breaks the deployment in CCv2.
+// https://github.com/SAP/spartacus/issues/5886
+if (environment.occBaseUrl && occConfig && occConfig.backend && occConfig.backend.occ) {
+  occConfig.backend.occ.baseUrl = environment.occBaseUrl;
+}
 
 @NgModule({
   declarations: [],
   imports: [
   ],
   providers: [provideConfig(layoutConfig), provideConfig(mediaConfig), ...defaultCmsContentProviders, provideConfig(<OccConfig>{
-    backend: {
-      occ: {
-      }
-    },
+    backend: occConfig.backend,
   }), provideConfig(<SiteContextConfig>{
     context: {
+      // TODO: adjust parameters and site
+      // urlParameters: ['baseSite', 'language', 'currency'],
+      // baseSite: ['electronics-spa'],
       currency: ['USD'],
       language: ['en'],
     },
